astropy/units/physical.py

```diff
@@ -465,15 +467,17 @@ def get_physical_type(obj):
 
     Parameters
     ----------
-    obj : `~astropy.units.UnitBase`, `str`, number, or `~astropy.units.Quantity`
-        The object for which to retreive the physical type.  This object
-        should be a unit, the name of a physical type, the number one, a
-        `~astropy.units.Quantity`, or an `object` that has a ``unit``
-        attribute that is a unit.
+    obj
```

Comment:
when #11118 is merged, this can be 
```suggestion
    obj : quantity-like or `~astropy.units.PhysicalType`-like
```

Reply:
Good idea!  Is there any problem with merging this pre-emptively?

Reply:
Should be no problem to just do this already - current sphinx would just render if a little differently.